plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1777,4 +1899,37 @@ private Session broadcastJoinDistribution(boolean dynamicFilteringEnabled)
                 .setSystemProperty(ENABLE_DYNAMIC_FILTERING, Boolean.toString(dynamicFilteringEnabled))
                 .build();
     }
+
+    protected String getTableLocation(String tableName)
+    {
+        return (String) computeScalar("SELECT DISTINCT regexp_replace(\"$path\", '/[^/]*$', '') FROM " + tableName);
+    }
+
+    private String getTableComment(String tableName)
+    {
+        return (String) computeScalar(format(
+                "SELECT comment FROM system.metadata.table_comments WHERE catalog_name = '%s' AND schema_name = '%s' AND table_name = '%s'",
+                getSession().getCatalog().orElseThrow(),
+                SCHEMA,
+                tableName));
+    }
+
+    private String getColumnComment(String tableName, String columnName)
+    {
+        return (String) computeScalar(format(
+                "SELECT comment FROM information_schema.columns WHERE table_schema = '%s' AND table_name = '%s' AND column_name = '%s'",
+                SCHEMA,
+                tableName,
+                columnName));
+    }
+
+    private void dropTableFromMetastore(String tableName)
+    {
+        HiveMetastore metastore = new BridgingHiveMetastore(
```

Comment:
I would move to `HiveMetastore metastore` to fields. Initialization the object per call is redundant in my opinion.

Reply:
Addressed!